src/spec/verifiable-payid-protocol-NC.md

```diff
@@ -0,0 +1,379 @@
+                  "typ": "JOSE+JSON",
+                  "b64": "false",
+                  "crit": ["b64"],
+                  "jwk": {
+                      "kty": "EC",
+                      "use": "sig",
+                      "crv": "secp256k1", 
+                      "x": "b8w36l6eCf7GyD5fvXp0Xj7ugdFuvYYcnmb1VRjBl5g=", 
+                      "y": "Tp8RPAf4dWkd+K/BApSW/Ey5UJs53NOPJRqDNZzItPc=",
+                  },
+                },
+                "signature": "base64Signature",
+              }
+              ]
+            "payload": {
+                "exp" : 1234574940,
+                "payId": "bob$wallet.com",
+                "payIdAddress": {
+                  "expTime": 34874613475,
+                  "paymentNetwork": "XRPL",
+                  "environment": "TESTNET",
+                  "addressDetailsType": "CryptoAddressDetails",
+                  "addressDetails": {
+                    "address": "T7CKYKhRujaxEs9fSxQwJApHsQVPKUgD7EtLWCGTAFBwTha"
+                   }
+                 }
+              }
+            }
+          ]
+      }
+
+ In the above example, the `PaymentInformation` resource is a pre-signed message with the PayID private keys of the PayID owner Bob. Bob's non-custodial wallet retrieves this response and sends it to the PayID client.
+
+# Security Considerations
+  This security considerations section only considers verifiable PayID clients and servers bound to implementations as defined in this document.
+
+  The security guarantees mentioned in [PAYID-PROTOCOL][] apply to this protocol. In this section, we discuss a security model for the Verifiable PayID protocol for non-custodial service providers.
+
+## Security Model for Non-Custodial PayID Service Providers
+
+  In the current security model, non-custodial wallets do not store their customers’ keys.  Instead, wallet customers hold their private keys on their own device(s). There is a no trust requirement between the service provided by a non-custodial wallets and its customers. Because customers in this scenario hold the private keys:
+  * Wallets are not liable for any consequences coming from the loss, compromise or theft of customers' private keys.
+  * The Non-custodial wallets do not require their customers to trust their servers in case wallets servers go malicious or are compromised.
+
+  This extension of Basic PayID protocol preserves this trust model. Rather than requiring the PayID server to provide accurate PayID response for their customers, the PayID owners can generate these signed mappings with their own PayID private key locally on their app/device. The sender of the payment (PayID client wallet’s customer) can easily verify these signatures out-of-band with the receiver (i.e., PayID owner). This eliminates any risk of the non-custodial PayID server wallet losing its private keys, going malicious, getting hacked, or becoming otherwise compromised in a way that customers might lose funds.
+
+## Using JWTs
```

Comment:
```suggestion
## Using JWS
```

Not really sure what this heading title should be, but I don't think JWT is used in this spec -- only JWS.